pkg/machines/components/diskAdd.jsx

```diff
@@ -381,8 +386,13 @@ class AddDiskModalBody extends React.Component {
 
     getDefaultVolumeFormat(pool) {
         // For the valid volume format types for different pool types see https://libvirt.org/storage.html
+        if (['disk'].indexOf(pool.type) > -1)
+            return 'none';
+
         if (['dir', 'fs', 'netfs', 'gluster', 'vstorage'].indexOf(pool.type) > -1)
             return 'qcow2';
+
```

Comment:
Documentation mentions that volume of pool type RDB can have only (thus defaultly too) "raw" as volume format (https://libvirt.org/storage.html). So maybe it could be useful to reflect it in this function too?

Reply:
Raw is the default format and we don't set it here at all, so I think RBD backend is fine here.

Reply:
Oh right, it seems that libvirt will fill it defaulty on other pool types if we do not pass format type.